src/go/rpk/pkg/config/params.go

```diff
@@ -133,6 +133,8 @@ const (
 	xAdminClientKey  = "admin.tls.client_key_path"
 )
 
+const DefaultConfigPath = "/etc/redpanda/redpanda.yaml"
```

Comment:
nit: I think this could just be `DefaultPath` since the package is already `config`